src/vs/workbench/browser/parts/editor/textResourceEditor.ts

```diff
@@ -140,15 +140,13 @@ export class AbstractTextResourceEditor extends BaseTextEditor {
 	 * This allows users to click on the output panel to stop scrolling when they see something of interest.
 	 * To resume, they should scroll to the end of the output panel again.
 	 */
-	revealLastLine(smart: boolean): void {
+	revealLastLine(smart): void {
```

Comment:
The "smart" behavior was taken out of this function. Instread, the cursor position affects the state of the lock.

Reply:
I left the parameter by mistake and fixed it in the second commit.